core/unix/include/sigcontext.h

```diff
@@ -341,8 +360,8 @@ typedef struct _kernel_sigcontext_t {
 /*
  * Header to be used at the beginning of structures extending the user
  * context. Such structures must be placed after the rt_sigframe on the stack
- * and be 16-byte aligned. The last structure must be a dummy one with the
- * magic and size set to 0.
+ * and be 16-byte aligned. The last structure must be a null terminator context
+ * with the magic and size set to 0.
  */
 struct _aarch64_ctx {
     __u32 magic;
```

Comment:
Comment that this holds a *_MAGIC value that identifies what this is: one of the structures below.

Reply:
Done.